test/built-ins/Temporal/ZonedDateTime/prototype/toString/negative-zoneddatetime-rounding.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.zoneddatetime.prototype.tostring
+description: >
+  RoundNumberToIncrementAsIfPositive should multiply the remainder by an extra sign
+  before comparing it.
+features: [Temporal]
+---*/
+
+const instance = new Temporal.ZonedDateTime(-999999999999999990n, "UTC"); // '1938-04-24T22:13:20.00000001+00:00[UTC]'
+const roundedDown = "1938-04-24T22:13:20.000+00:00[UTC]";
+const roundedUp = "1938-04-24T22:13:20.001+00:00[UTC]";
+
+assert.sameValue(
+    instance.toString({smallestUnit: 'millisecond', roundingMode: 'halfCeil'}),
```

Comment:
Same about indentation

Reply:
Fixed in [d5d1f2a](https://github.com/tc39/test262/pull/4624/commits/d5d1f2a32df8092c26db0d4fc358d5c76129f928)